bchrpc/server.go

```diff
@@ -2929,8 +2929,7 @@ func (s *GrpcServer) getSlpToken(hash *chainhash.Hash, vout uint32, scriptPubKey
 // NOTE: this is launched as a goroutine and does not return errors!
 //
 func (s *GrpcServer) slpEventHandler() {
-
-	if s.slpIndex == nil {
+	if s.slpIndex == nil || s.txIndex == nil {
```

Comment:
Could be explanded to include ` ... || !s.slpIndex.GraphSearchEnabled()`, and then remove the two gs checks below.

Reply:
Errt, ignore that comment. 

The code should actually only be `if s.slpIndex == nil`.  We don't want the check for `s.txIndex == nil`.

Reply:
Fixed!